config/detekt/detekt-baseline.xml

```diff
@@ -11,5 +11,14 @@
     <ManuallySuppressedIssues>
     </ManuallySuppressedIssues>
     <CurrentIssues>
+        <ID>ForbiddenComment:Logger.kt$Logger$// TODO: 14/02/2019 The idea is for this class to have a system where we can send a stream of logs to the merchant</ID>
```

Comment:
Is there another way to do this? I'm wondering what happens if we move a class that has one of these comments, or if we edit/delete a comment.

Reply:
Makes sense